[BUG] Link checker takes some valid links as broken

**Describe the bug**
In #1620 issue Link Checker takes a number of working links as broken or invalid

**To Reproduce**
Steps to reproduce the behavior:
1. Go to ' #1620 '
2. Check all the [TIMEOUT], [ERROR] only https://app.santiment.net/ and https://nu.fi/, [202], [520], [500]


**Expected behavior**
All the listed above links looks good and valid

**Screenshots**
If applicable, add screenshots to help explain your problem.

**Device (please complete the following information if this is addressing the Chain.Love website):**
 - Type: [e.g. Mobile, Desktop, Tablet, etc.]
 - OS: [e.g. iOS]
 - Browser [e.g. chrome, safari]
 - Version [e.g. 22]

**Additional context**
Add any other context about the problem here.

--- a/.github/workflows/link-check-analysis.yaml
+++ b/.github/workflows/link-check-analysis.yaml
@@ -144,6 +144,7 @@ jobs:
           failIfEmpty: false
           args: >
             --extensions csv
+            --method GET
             --accept 200,204,400,401,403,405,429
             --timeout 5
             --no-progress

